app/MMB/src/components/Separators.js

```diff
@@ -11,8 +11,8 @@ import { Color, StyleSheet } from '@kiwicom/mobile-shared';
  * |---------------------------|
  * |                           |
  */
-export function SeparatorFullWidth() {
-  return <View style={styleSheet.separator} />;
+export function SeparatorFullWidth({ style }: { style?: StylePropType }) {
+  return <View style={[styleSheet.separator, style]} />;
```

Comment:
First of all, I don't like this change. You can just wrap your `SeparatorFullWidth` in a `View` if you need it. No need for another property... 🙂 But more importantly, the FLow type should be exact. Same for `Row` component.